scripts/record-cluster-fixtures.ts

```diff
@@ -0,0 +1,158 @@
+            fetch(url, {
+                body: JSON.stringify({
+                    id: 1,
+                    jsonrpc: '2.0',
+                    method: 'getEpochInfo',
+                }),
+                headers: { 'Content-Type': 'application/json' },
+                method: 'POST',
+            }),
+            fetch(url, {
+                body: JSON.stringify({
+                    id: 2,
+                    jsonrpc: '2.0',
+                    method: 'getEpochSchedule',
+                }),
+                headers: { 'Content-Type': 'application/json' },
+                method: 'POST',
+            }),
+            fetch(url, {
+                body: JSON.stringify({
+                    id: 3,
+                    jsonrpc: '2.0',
+                    method: 'getFirstAvailableBlock',
+                }),
+                headers: { 'Content-Type': 'application/json' },
+                method: 'POST',
+            }),
+        ]);
+
+        const [epochInfoData, epochScheduleData, firstBlockData] = await Promise.all([
+            epochInfoResp.json(),
+            epochScheduleResp.json(),
+            firstBlockResp.json(),
+        ]);
+
+        // Check for errors
+        if (epochInfoData.error) throw new Error(`getEpochInfo error: ${JSON.stringify(epochInfoData.error)}`);
+        if (epochScheduleData.error)
+            throw new Error(`getEpochSchedule error: ${JSON.stringify(epochScheduleData.error)}`);
+        if (firstBlockData.error)
+            throw new Error(`getFirstAvailableBlock error: ${JSON.stringify(firstBlockData.error)}`);
+
+        // Build fixture data
+        const fixtureData = {
+            epochInfo: {
+                absoluteSlot: BigInt(epochInfoData.result.absoluteSlot),
+                blockHeight: BigInt(epochInfoData.result.blockHeight),
+                epoch: BigInt(epochInfoData.result.epoch),
+                slotIndex: BigInt(epochInfoData.result.slotIndex),
+                slotsInEpoch: BigInt(epochInfoData.result.slotsInEpoch),
+            },
+            epochSchedule: {
+                firstNormalEpoch: BigInt(epochScheduleData.result.firstNormalEpoch),
+                firstNormalSlot: BigInt(epochScheduleData.result.firstNormalSlot),
+                leaderScheduleSlotOffset: BigInt(epochScheduleData.result.leaderScheduleSlotOffset),
+                slotsPerEpoch: BigInt(epochScheduleData.result.slotsPerEpoch),
+                warmup: epochScheduleData.result.warmup,
+            },
+            firstAvailableBlock: BigInt(firstBlockData.result),
+        };
+
+        saveFixture(fixtureKey, fixtureData);
+
+        console.log(`   ✨ Success!`);
+    } catch (error) {
+        console.error(`   ❌ Failed:`, error);
+        throw error;
+    }
+}
+
+// Common test configurations
+const TEST_CONFIGS = {
+    'default-test': {
+        cluster: 'mainnet-beta',
+        url: 'https://api.mainnet-beta.solana.com',
```

Comment:
Let's use RPC addresses from cluster as well to. Otherwise this may drift upon changes into main RPCs